src/caffe/test/test_gradient_based_solver.cpp

```diff
@@ -61,8 +62,6 @@ class GradientBasedSolverTest : public MultiDeviceTest<TypeParam> {
     ostringstream proto;
     proto <<
        "max_iter: " << num_iters << " "
-       "base_lr: " << learning_rate << " "
-       "lr_policy: 'fixed' "
        "iter_size: " << iter_size << " "
```

Comment:
Addressed in https://github.com/matthiasplappert/caffe/commit/322a9de77b6d56f4dea0fa060bc02e7b2c7b97d5